internal/sanitize/sanitize.go

```diff
@@ -66,7 +66,24 @@ func (q *Query) Sanitize(args ...any) (string, error) {
 
 			// Prevent SQL injection via Line Comment Creation
 			// https://github.com/jackc/pgx/security/advisories/GHSA-m7wr-2xf7-cm9p
-			str = "(" + str + ")"
+			// If the value starts with a dash then we need to put parens around it
+			if strings.HasPrefix(str, "-") {
+				str = "(" + str + ")"
+			}
+
+			// Convert problematic line breaks to escape form
+			// See: https://www.postgresql.org/docs/9.0/sql-syntax-lexical.html Table 4.1
+			if strings.ContainsAny(str, "\b\f\n\r\t") {
```

Comment:
In the scenario where the value has both a line break AND a - prefix, we get an invalid `E('value')` instead of `(E'value')`